### PR TITLE
feat(jobboard): add job section to homepage - fixes #66

### LINKED ISSUE #66: implement jobboard on the homepage!
A jobboard should be implemented on the homepage.
The jobboard should contain:
- company logo
- job title
- short job description
- to be put beneath organisers

&  link to job description entry on 3th party website

- [x] design entry
- [ ] put include on every page of the website 
- [ ] link to job descriptions on 3th party website

### PR DESCRIPTION
**Pull request by SamVerschueren**
First draft of the job section. If you want some things differently, let me know. Also not sure about the grid classes I used. Not very familiair with them to be honest. So any help or improvements on those are more then welcome as well.

I hope this is what you where looking for. Feel free to say so if not, you don't have to merge because of my beautiful eyes. I can handle rejection ;).

### CODE REVIEW

File: src/components/jobboard/_build/jobboard.less [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  background: white;
+
+  h1 {
+    text-align: center;
+    margin-bottom: 30px;
+  }
+
+}
+
+.community-logo {
```
Comment (samvloeberghs) [inline src/components/jobboard/_build/jobboard.less]: Bad naming? Any specific reason why community?

**Review by samvloeberghs — CHANGES_REQUESTED** [review]

**Comment by samvloeberghs** [comment]
Thx Sam! Will look into the design tomorrow!

**Comment by samvloeberghs** [comment]
@SamVerschueren as you see I've updated some parts of the grid, to also cover the mobile view better (1colom): I think it just needs the update of the naming like mentioned before.. nice job, thanks alot for this! :)

**Comment by samvloeberghs** [comment]
what we still need to agree on is where to put it exactly on the website. For now I would put it on every page and where you originally put it on the page

File: src/components/jobboard/_build/jobboard.less (hunk 1 above)
Comment (SamVerschueren) [reply src/components/jobboard/_build/jobboard.less]: It was a copy paste from the [sponsorcommunity stylesheet](https://github.com/ng-be/www-ng-be-org/blob/master/src/components/sponsorscommunity/_build/sponsorscommunity.less#L38). I agree, fixed it.

**Comment by SamVerschueren** [comment]
I want to add it on every page, but wouldn't it clutter every page? Maybe we could add a `JOBS` page and show that information again instead? I'm just worried that we will have too much side-content on one page which is not associated directly with the main content of the page.

**Comment by samvloeberghs** [comment]
@SamVerschueren let's consider that a different task!

**Review by samvloeberghs — APPROVED** [review]

**Pull request merged**